filter.imp

```diff
@@ -1,17 +1,20 @@
 [
    { include: ["@.asm-generic/unaligned.h.", private, "<asm/unaligned.h>", "public"] },
-   { include: ["@.asm-generic/errno.h.", private, "<asm/errno.h>", "public"] },
+   { include: ["@.asm-generic/errno.h.", private, "<linux/errno.h>", "public"] },
+   { include: ["@.asm-generic/errno-base.h.", private, "<linux/errno.h>", "public"] },
+   { include: ["@.asm/errno.h.", private, "<linux/errno.h>", "public"] },
+   { include: ["@.asm/bug.h.", private, "<linux/bug.h>", "public"] },
```

Comment:
bug.h appears in include/asm-generic/Kbuild.  I would like to see those separated out into a separate block of these rules in this with a comment describing the purpose of include/asm-generic/Kbuild, at some point BEFORE you resend a v2 of your kernel patch series.